modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/chemistry_for_ERP.dm

```diff
@@ -297,17 +354,42 @@
 /datum/reagent/drug/aphrodisiac/breast_enlarger/overdose_effects(mob/living/carbon/human/exposed_mob) //Turns you into a female if character is male. Also supposed to add breasts but enlargement_amount'm too dumb to figure out how to make it work
 	var/obj/item/organ/genital/penis/mob_penis = exposed_mob.getorganslot(ORGAN_SLOT_PENIS)
 	var/obj/item/organ/genital/testicles/mob_testicles = exposed_mob.getorganslot(ORGAN_SLOT_TESTICLES)
+	/// Makes above comment actually work.
+	if(!exposed_mob.getorganslot(ORGAN_SLOT_BREASTS))
```

Comment:
Use an early return.

Reply:
where?

Reply:
if(!exposed_mob.getorganslot(ORGAN_SLOT_BREASTS)
  return

Reply:
but uhh why

Reply:
as expected, the code below that does not run
I don't understand the point of putting an early return there, since it's specifically for ODing on the breast enlargement chem, and is supposed to insert the breasts organ into the mob if they don't have breasts.